impress-org/givewp
# The Donor Dashboard shortcode now works on 4.9.x

Resolves #5738 

## Description

This PR introduces changes to ensure that the Donor Dashboard shortcode works as expected with WP core 4.9.x, and that the generated Donor Dashboard page uses a shortcode when the block editor is not enabled.

## Affects

This PR affects frontend rendering logic of the Donor Dashboard application, removing all WP 5.0 dependencies. Additionally, this PR affects the Donor Dashboard page generation logic, specifically adding conditions to check for whether or not the block editor is enabled at the point the Donor Dashboard page is being generated.

## Visuals

N/A

## Testing Instructions

On WP 4.9.x
1. Generate a Donor Dashboard page
2. Check that it works as expected on the frontend
3. Check that the page uses a shortcode in the editor to display the Donor Dashboard

On WP 5.x with Classic Editor enabled
1. Generate a Donor Dashboard page
2. Check that it works as expected on the frontend
3. Check that the page uses a shortcode in the editor to display the Donor Dashboard

On WP 4.9.x without Gutenberg plugin enabled
1. Generate a Donor Dashboard page
2. Check that it works as expected on the frontend
3. Check that the page uses a block in the editor to display the Donor Dashboard

On WP 5.x without Classic Editor enabled
1. Generate a Donor Dashboard page
2. Check that it works as expected on the frontend
3. Check that the page uses a block in the editor to display the Donor Dashboard
4. 
## Pre-review Checklist
